server/container/guice/memory-guice/pom.xml

```diff
@@ -225,6 +237,50 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>docker packaging</id>
+                        <goals>
+                            <goal>dockerBuild</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <to>
+                                <image>james-server-memory:${buildId}</image>
+                            </to>
+                            <container>
+                                <mainClass>org.apache.james.MemoryJamesServerMain</mainClass>
+                                <volumes>
+                                    <volume>/root/conf</volume>
+                                    <volume>/logs</volume>
+                                </volumes>
+                                <jvmFlags>
+                                    <jvmFlag>-Dworking.directory=/root</jvmFlag>
+                                    <jvmFlag>-Dlogback.configurationFile=/root/conf/logback.xml</jvmFlag>
+                                </jvmFlags>
+                                <ports>
+                                    <port>25</port>
+                                    <port>80</port>
+                                    <port>143</port>
+                                    <port>465</port>
+                                    <port>587</port>
+                                    <port>993</port>
```

Comment:
We don't expose port `4000` for glowroot ?

Reply:
As it's not started by default, I did not. But if you have another opinion, let me know.

Reply:
Ok, it's good for me then.